<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinDialog</class>
 <widget class="QDialog" name="JoinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DailyFlow - 회원가입</string>
  </property>
  <property name="styleSheet">
   <string>QDialog {
    background-color: #f5f5f5;
}

QGroupBox {
    background-color: white;
    border: 2px solid #2196F3;
    border-radius: 12px;
    margin-top: 20px;
    padding: 25px;
    font-size: 18px;
    font-weight: bold;
    color: #2196F3;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 20px;
    padding: 0 10px;
}

QLabel {
    font-size: 13px;
    font-weight: bold;
    color: #333;
    min-width: 120px;
}

QLineEdit, QDateEdit {
    padding: 8px 12px;
    border: 2px solid #e0e0e0;
    border-radius: 6px;
    background-color: white;
    font-size: 13px;
}

QLineEdit:focus, QDateEdit:focus {
    border: 2px solid #2196F3;
}

QDateEdit::drop-down {
    border: none;
    width: 25px;
}

QDateEdit::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 4px solid #666;
    margin-right: 8px;
}

QComboBox {
    padding: 8px 12px;
    border: 2px solid #e0e0e0;
    border-radius: 6px;
    background-color: white;
    font-size: 13px;
}

QComboBox:focus {
    border: 2px solid #2196F3;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 4px solid #666;
    margin-right: 8px;
}

QPushButton {
    background-color: #2196F3;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-weight: bold;
    font-size: 14px;
    min-width: 100px;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #1565C0;
}

QPushButton#cancelButton {
    background-color: #757575;
}

QPushButton#cancelButton:hover {
    background-color: #616161;
}

QPushButton#cancelButton:pressed {
    background-color: #424242;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QGroupBox" name="joinGroupBox">
     <property name="title">
      <string>회원가입</string>
     </property>
     <layout class="QVBoxLayout" name="groupBoxLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="accountSectionLabel">
        <property name="styleSheet">
         <string>font-size: 15px; color: #2196F3; font-weight: bold; margin-top: 5px;</string>
        </property>
        <property name="text">
         <string> 계정 정보</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="usernameLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>아이디</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="placeholderText">
           <string>아이디를 입력하세요</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="passwordLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>비밀번호</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>비밀번호를 입력하세요</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="confirmPasswordLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="confirmPasswordLabel">
          <property name="text">
           <string>비밀번호 확인</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>비밀번호를 다시 입력하세요</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string>background-color: #e0e0e0;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="personalSectionLabel">
        <property name="styleSheet">
         <string>font-size: 15px; color: #2196F3; font-weight: bold; margin-top: 5px;</string>
        </property>
        <property name="text">
         <string> 개인 정보</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="nameLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>이름</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLineEdit">
          <property name="placeholderText">
           <string>이름을 입력하세요</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="emailLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>이메일</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailLineEdit">
          <property name="placeholderText">
           <string>이메일</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="emailDomainLabel">
          <property name="styleSheet">
           <string>font-size: 14px; min-width: 20px;</string>
          </property>
          <property name="text">
           <string>@</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="emailDomainComboBox">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>naver.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gmail.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>daum.net</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kakao.com</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="dateOfBirthLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="dateOfBirthLabel">
          <property name="text">
           <string>생년월일</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateOfBirthEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="addressLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="addressLabel">
          <property name="text">
           <string>주소</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addressLineEdit">
          <property name="placeholderText">
           <string>주소를 입력하세요</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="addressDetailLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="addressDetailLabel">
          <property name="text">
           <string>상세 주소</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addressDetailLineEdit">
          <property name="placeholderText">
           <string>상세 주소를 입력하세요</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="minimumSize">
           <size>
            <width>148</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>가입하기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="minimumSize">
           <size>
            <width>148</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>취소</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
